set.mm

```diff
@@ -507326,6 +507696,61 @@ codes is an increasing chain (with respect to inclusion).  (Contributed
       NUIUABJKUGUHOPQFERT $.
   $}
 
+  ${
+    $d A a $.  $d B a $.  $d M a $.  $d V a $.
+    $( The wff ` ( A e. B -/\ B e. A ) ` encoded as ` ( ( A e.g B ) `
+       ` |g ( B e.g A ) ) ` is true in any model ` M ` .  This is the model
+       theoretic proof of ~ elnanel .  (Contributed by AV, 5-Nov-2023.) $)
+    elnanelprv $p |- ( ( M e. V /\ A e. _om /\ B e. _om )
+                       -> M |= ( ( A e.g B ) |g ( B e.g A ) ) ) $=
+      ( va wcel com w3a cgoe co cgna cprv wbr csate cmap wceq cfv wn wa cvv a1i
+      cv crab simp1 3simpc pm3.22 3adant1 eqid satefvfmla1 syl3anc wnan elnanel
+      wo nanor mpbi rabeqc syl6eq wb ovex prv sylancl mpbird ) CDFZAGFZBGFZHZCA
+      BIJZBAIJZKJZLMZCVINJZCGOJZPZVFVKAEUBZQZBVNQZFZRVPVOFZRUMZEVLUCZVLVFVCVDVE
+      SVEVDSZVKVTPVCVDVEUDZVCVDVEUEVDVEWAVCVDVEUFUGABBACDVIEVIUHUIUJVSEVLVSVNVL
+      FVQVRUKVSVOVPULVQVRUNUOUAUPUQVFVCVITFVJVMURWBVGVHKUSVICDTUTVAVB $.
+  $}
+
+  ${
+    $d U x $.
+    $( Every wff encoded as ` U ` is true in an empty model.  (Contributed by
+       AV, 19-Nov-2023.) $)
+    prv0 $p |- ( U e. ( Fmla ` _om ) -> (/) |= U ) $=
+      ( vx com cfmla cfv wcel c0 cprv wbr csate co wceq csat sate0 cv wn peano1
+      wa cvv 0ex wal wf n0ii intnan a1i f00 sylnibr pm3.2i satfvel mp3an1 mtand
+      alrimiv eq0 sylibr eqtrd cmap prv mpan wne ne0ii map0b mp1i eqeq2d mpbird
+      wb bitrd ) ACDEZFZGAHIZGAJKZGLZVHVJACGGMKEEZGAVGNVHBOZVLFZPZBUAVLGLVHVOBV
+      HVNCGVMUBZVHVMGLZCGLZRZVPVSPVHVRVQGCQUCUDUECVMUFUGGSFZVTRVHVNVPVTVTTTUHVM
+      AGGSSUIUJUKULBVLUMUNUOVHVIVJGCUPKZLZVKVTVHVIWBVETAGSVGUQURVHWAGVJCGUSWAGL
+      VHGCQUTCVAVBVCVFVD $.
+  $}
```

Comment:
The set `U` does not seem to be special, so I think the comment should rather be: _every wff is true in an empty model_?

Reply:
The formula `E. x T.` is certainly not true in the empty model, so either there is a contradiction in set.mm, or, more probably, the comments do not describe the associated proofs/definitions correctly.

Reply:
Ok, so after a quick look at df-prv, we see that it is not adapted to empty models. So this limitation should be indicated there, and after that, probably no use of the phrase "empty model" should be made. (The statement can stay, but of course it doesn't mean anything for the "empty model".) A comment could be:

Since `|=` is defined in terms of the interpretations making the given formula true, it is not defined on the empty model, since there, there are no interpretations. In particular, the empty set on the LHS `|=` should not be interpreted as the empty model.  Statement `~ prv0` shows that our definition  yields `(/) |= U` for all formulas, though of course the formula `E. x x = x` is not satisfied on the empty model.

Reply:
I will enhance the comments of df-prv and prv0 accordingly.